package.json

```diff
@@ -36,26 +36,31 @@
     "eslint": "eslint .",
     "test": "mocha",
     "jsdoc": "jsdoc lib/* -d dist/doc",
-    "travis": "npm run lint && npm run test"
+    "travis": "npm run lint && npm run test",
+    "browserify": "mkdirp dist && node_modules/browserify/bin/cmd.js lib/index.js -o dist/pagexray.js --standalone PageXray -p [ browserify-banner --template 'PageXray v<%= pkg.version %> - Xray your HAR file and know all about the page - https://github.com/sitespeedio/pagexray' ]",
```

Comment:
It works. Just a bit confusing maybe that ```npm run browserify``` would run the executable from the browserify module if the script was named something else, i.e. it's shadowing. But now it runs the script, that in turn runs browserify. I didn't know before that node_modules/.bin/ was added to the PATH inside npm scripts, so that's a nice way to get around that issue.